script/DeployPolygon.s.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {DeployCREATE3Factory} from './DeployCREATE3Factory.s.sol';
+import {DeployRouter} from './DeployRouter.s.sol';
+import {DeployAaveV3FlashLoanCallback} from './callbacks/DeployAaveV3FlashLoanCallback.s.sol';
+import {DeployBalancerV2FlashLoanCallback} from './callbacks/DeployBalancerV2FlashLoanCallback.s.sol';
+
+contract DeployPolygon is
+    DeployCREATE3Factory,
+    DeployRouter,
+    DeployAaveV3FlashLoanCallback,
+    DeployBalancerV2FlashLoanCallback
+{
+    /// @notice Set up deploy parameters and deploy contracts whose `deployedAddress` equals `UNDEPLOYED`.
+    function setUp() external {
+        create3FactoryConfig = Create3FactoryConfig({
+            deployedAddress: 0x2a36F87b2Ec3dE23617907461aa3DA0cC4bc3f1f,
+            deployer: 0xa3C1C91403F0026b9dd086882aDbC8Cdbc3b3cfB
```

Comment:
Just want to confirm that we use testing account as deployer and other roles like pauser.

Reply:
Yes, it's expected since the create3 factory is also used for testing purposes.